lib/zendesk_apps_tools/command.rb

```diff
@@ -118,6 +118,36 @@ def package
       true
     end
 
+    desc 'merge_markdown', 'Merge .md marketplace content files into en.json'
+    method_option :overwrite, required: false, aliases: '-o'
+    def merge_markdown 
+      if Dir["#{app_dir}/translations/en.json"].empty?
+        say_status 'merge_markdown', "Please create a translations/en.json first", :red
+        return false
+      end
+      
+      translation_files = Dir["#{app_dir}/translations/{long_description,installation_instructions}.md"]
+      en_json = JSON.parse(File.read("#{app_dir}/translations/en.json"))
+
+      translation_files.each do |file_name|
+        translation_key = File.basename(file_name, '.md')
+        say_status 'merge_markdown', "Reading #{translation_key}.md"
+        unless en_json['app'][translation_key].to_s.empty?
+          if options[:overwrite]
+            say_status 'merge_markdown', "Overwriting the current value of #{translation_key} in en.json", :yellow
+          else
+            say_status 'merge_markdown', "You already have a value for #{translation_key} in en.json. Please remove it or use the -o flag", :red
+            return false
+          end
+        end
+        en_json['app'][translation_key] = File.read(file_name)
+      end
+
+      say_status 'merge_markdown', "Writing to en.json"
+      File.write("#{app_dir}/translations/en.json", en_json.to_json)
```

Comment:
So `en.json` is *not* the source of truth for zendesk apps, `en.yml` is. If anybody @ zendesk would use this, it would cause problems.

However for external devs `en.json` is the source of truth, so that will work fine.

Can you make it so that it checks that if a `en.yml` is present, it writes it to `en.yml`?

Reply:
I don't want to make it too magical. Would you be OK with adding a flag instead to write to a yml file?

Reply:
@ocke I don't want to do this. The formatting differences between en.json and the yml files make the code really fiddly really quickly. If you're using a yml file you don't need this utility at all, since you can just type markdown with real newlines into a yml file with no problems (see the Text app).

Reply:
Discussed in 🍖. I'm gonna leave it out.